core-bundle/src/Resources/contao/dca/tl_files.php

```diff
@@ -607,8 +607,9 @@ public function addFileLocation($value, DataContainer $dc)
 	public function checkFilename($varValue, DataContainer $dc)
 	{
 		$varValue = str_replace('"', '', $varValue);
+		$varValue = trim($varValue, '/');
```

Comment:
I think we should probably split the value here on `/` and validate every single part with the code below (not allowing `.` at the and and checking maxlength)

Reply:
Changed in 2c037b1e716c692bc469ac15b48eaac617b766c9.

Reply:
The maxlength check is still on the whole path, but it should be checked on every single chunk I think?

Reply:
Changed in 37e60da26f54a1f9c88dc333005e3eef0041be95.